code-samples/serving/hello-world/helloworld-java-spring/README.md

```diff
@@ -133,101 +133,100 @@ During the creation of a Service, Knative performs the following steps:
-    into the file:
-
-        ```yaml
-        apiVersion: serving.knative.dev/v1
-        kind: Service
-        metadata:
-          name: helloworld-java-spring
-          namespace: default
-        spec:
-          template:
-            spec:
-              containers:
-                - image: docker.io/{username}/helloworld-java-spring
-                  env:
-                    - name: TARGET
-                      value: "Spring Boot Sample v1"
-        ```
-        Where `{username}` is your Docker Hub username.
-
-        **Note:** Ensure that the container image value in `service.yaml` matches the container you built in the previous step.
+### YAML
+
+1. Create a new file named `service.yaml` and copy the following service definition
+   into the file:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: helloworld-java-spring
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+        - image: docker.io/{username}/helloworld-java-spring
+          env:
+            - name: TARGET
+              value: "Spring Boot Sample v1"
+```
+Where `{username}` is your Docker Hub username.
 
-    1. Apply the YAML file by running the command:
+**Note:** Ensure that the container image value in `service.yaml` matches the container you built in the previous step.
 
-        ```bash
-        kubectl apply -f service.yaml
-        ```
+1. Apply the YAML file by running the command:
 
-=== "kn"
+```bash
+kubectl apply -f service.yaml
+```
 
-       With `kn` you can deploy the service with
+### kn
 
-       ```bash
-       kn service create helloworld-java-spring --image=docker.io/{username}/helloworld-java-spring --env TARGET="Spring Boot Sample v1"
-       ```
+With `kn` you can deploy the service with
 
-       This will wait until your service is deployed and ready, and ultimately it will print the URL through which you can access the service.
+```bash
+kn service create helloworld-java-spring --image=docker.io/{username}/helloworld-java-spring --env TARGET="Spring Boot Sample v1"
+```
 
+This will wait until your service is deployed and ready, and ultimately it will print the URL through which you can access the service.
 
 ## Verification
 
 1. Find the domain URL for your service:
 
-    === "kubectl"
-           ```bash
-           kubectl get ksvc helloworld-java-spring  --output=custom-columns=NAME:.metadata.name,URL:.status.url
-           ```
+### kubectl
 
-           Example:
+```bash
+kubectl get ksvc helloworld-java-spring  --output=custom-columns=NAME:.metadata.name,URL:.status.url
+```
 
-           ```bash
-           NAME                      URL
-           helloworld-java-spring    http://helloworld-java-spring.default.1.2.3.4.xip.io
-           ```
+Example:
 
+```bash
+NAME                      URL
+helloworld-java-spring    http://helloworld-java-spring.default.1.2.3.4.xip.io
+```
 
-    === "kn"
+### kn
```

Comment:
```suggestion
    - For kn, run:
```

Reply:
I suggest removing the H3 heading here because these commands are part of a step.